lib/customers.js

```diff
@@ -85,6 +85,28 @@ function update (id, data, userToken, txId) {
     .then(camelize)
 }
 
+/**
+ * Update customer record
+ *
+ * @name updateCustomer
+ * @function
+ *
+ * @param {string} id Customer's id
+ * @param {object} data Fields to update
+ *
+ * @returns {Promise} Newly updated Customer
+ */
+async function updateCustomer (id, data) {
+  const formattedData = _.omit(['id'], _.mapKeys(_.snakeCase, data))
```

Comment:
we should be really careful with this and pick only the fields we want to update from data

Reply:
My idea was that the fields that we want to update are all of the ones that came in 'data', considering that it isn't always with all of the customer fields. Is it what you're saying?

Reply:
Right, what I'm saying is that you should use `_.pick` instead of `_.omit`.

The idea is to safeguard against future changes on data and only update the field when explicitly told to. So even if someone adds new stuff in data, we only update what we're passing to `_.pick`